serveradmin/serverdb/admin.py

```diff
@@ -31,12 +31,15 @@ class ServertypeAdmin(admin.ModelAdmin):
         ServertypeAttributeInline,
     )
 
-    def get_exclude(self, request, obj=None):
-        # Because of the great complexity when changing servertypes of existing
-        # objects and the little use-cases we have we just deny it for now.
+    def get_readonly_fields(self, request, obj=None):
         if obj:
+            # Because of the complexity when changing servertypes of existing
+            # objects and the little use-cases we have right now we don't
+            # support it.
             return ['ip_addr_type']
 
+        return super().get_readonly_fields(request, obj)
```

Comment:
You first want to get the value of the super method and then add your own on it.
``` python
def get_readonly_fields(self, request, obj=None):
    fields = super().get_readonly_fields(request, obj)
    #fields = super().get_readonly_fields(request, obj) or []
    if obj:
        fields.append('ip_addr_type')

    return fields
```

Reply:
Ah yes that makes sense 👍